distributions/JavaScript/trigger_docker_rebuild.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+curl -s -X POST -H "Content-Type: application/json" -H "Accept: application/json" -H "Travis-API-Version: 3" -H "Authorization: token $TRAVIS_TOKEN" -d '{ "request": { "branch":"master", "message": "Build Travis after changes in libamtrack/library" }}' https://api.travis-ci.com/repo/libamtrack%2FDockerFiles/requests
```

Comment:
This for now it is not working because it needs move DockerFile from travis.org to traivs.com

Reply:
Ive test it localy with my toekn and it works with travis.org, also Done